src/wspc/reader.py

```diff
@@ -79,12 +78,15 @@ def read_files_in_dir(dir_path):
 
     with os.scandir(dir_path) as entries:
         for entry in entries:
-            if entry.is_file() and entry.name.endswith(PATRIC_FILE_EXTENSION):
-                genome_id = entry.name.split(PATRIC_FILE_EXTENSION)[0]
-                pgfams = read_genome_file(entry)
-
-                genomes_ids.append(genome_id)
-                genomes_pgfams.append(pgfams)
+            if entry.is_file():
+                for extension, pgfam_col in PATRIC_FILE_EXTENSION_TO_PGFAM_COL.items():
+                    if entry.name.endswith(extension):
+                        genome_id = entry.name.split(extension)[0]
+                        pgfams = read_genome_file(entry, pgfam_col)
+
+                        genomes_ids.append(genome_id)
+                        genomes_pgfams.append(pgfams)
+                        break
```

Comment:
why is there a break here?

Reply:
The break is for the enternal loop (the loop in line 82). If we found the correct extension of the current entry, that we can stop go over the items in PATRIC_FILE_EXTENSION_TO_PGFAM_COL for this entry

Reply:
ah OK